src/screens/OrganizationActionItems/OrganizationActionItems.tsx

```
@@ -258,21 +258,47 @@ function organizationActionItems(): JSX.Element {
-                alt={displayName}
-              />
+            <div
+              className={styles.TableImage}
+              style={{ flexShrink: 0, marginRight: '8px' }}
+            >
+              {assignee?.avatarURL ? (
+                <img
+                  src={assignee.avatarURL}
+                  crossOrigin="anonymous"
+                  className={styles.TableImage}
+                />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)